mlos_bench/mlos_bench/tests/optimizers/conftest.py

```diff
@@ -154,3 +157,16 @@ def smac_opt_max(tunable_groups: TunableGroups) -> MlosCoreOptimizer:
             "max_ratio": 1.0,
         },
     )
+
+
+@pytest.fixture
+def manual_opt(tunable_groups: TunableGroups, mock_configs: List[dict]) -> ManualOptimizer:
+    """Test fixture for ManualOptimizer."""
+    return ManualOptimizer(
+        tunables=tunable_groups,
+        service=None,
+        config={
+            "max_suggestions": 2 * len(mock_configs),
```

Comment:
Exactly the sort of config annoyance I'm thinking to avoid.

Set `"max_cycles": 2` and let `max_suggestions = min(max_suggestions, max_cycles * len(cycle_tunable_values))` internally.